network-area-diagram-viewer/src/main/java/com/powsybl/ad/viewer/controller/ControllerDiagram.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.ad.viewer.controller;
+
+import com.powsybl.ad.viewer.model.DisplaySVG;
+import com.powsybl.ad.viewer.view.diagram.DiagramPane;
+import com.powsybl.ad.viewer.view.diagram.ContainerDiagramPane;
+import javafx.stage.Stage;
+
+import java.io.StringWriter;
+
+/**
+ * @author Louis Lhotte <louis.lhotte@student-cs.fr>
+ */
+public class ControllerDiagram
+{
+    private Stage primaryStage;
```

Comment:
You're not using a uniform style for your code, in fact there's a some rules in powsybl for that, which are described in a checkstyle config file: https://github.com/powsybl/powsybl-parent/tree/main/powsybl-build-tools/src/main/resources/powsybl-build-tools. I'll add the corresponding mvn plugin to this repo to check it automatically. Let's keep that esthetics issue for the end.